app/developers/deliver_routes.py

```diff
@@ -921,86 +877,63 @@ def collection_confirmation(submission_id: UUID) -> ResponseReturnValue:
     )
 
 
-@developers_deliver_blueprint.route("/submissions/<uuid:submission_id>/<uuid:question_id>", methods=["GET", "POST"])
-@is_platform_admin
-@auto_commit_after_request
-def ask_a_question(submission_id: UUID, question_id: UUID) -> ResponseReturnValue:
-    submission_helper = SubmissionHelper.load(submission_id)
-    question = submission_helper.get_question(question_id)
+class DGFFormRunner(FormRunner):
```

Comment:
nit: I wouldn't be massively upset with these subclasses just being defined in the `collections/runner` file, so that 'all' of the runners and runner config (url maps) live there?